src/components/Layout/index.js

```diff
@@ -13,6 +13,14 @@ class Layout extends Component {
     openMenu: false
   }
 
+  componentWillMount () {

```

Comment:
This is not a good idea since will not open the drawer on resize. I would suggest you and @JesseRWeigel to implement [Material-UI Responsive Drawer](https://material-ui-next.com/demos/drawers/#responsive-drawer) instead. What that does is on big size screens the drawer is open, and on small screen size the drawer is hidden and can be spawned via the menu button.

Reply:
Great Idea I am making this now @SebastianCApostolescu